file-formats/siad/type/zif_aff_siad_v1.intf.abap

```diff
@@ -0,0 +1,43 @@
+INTERFACE zif_aff_siad_v1
+  PUBLIC.
+
+  TYPES:
+
+    "! <p class="shorttext">Assignment</p>
+    "! Assignment
+    BEGIN OF ty_assignment,
+
+      "! <p class="shorttext">Business Role Template ID</p>
+      "! Business role template ID
+      "! $required
+      business_role_template_id   TYPE c LENGTH 30,
+
+      "! <p class="shorttext">Launchpad Space Template ID</p>
+      "! Launchpad space template ID
+      "! $required
+      launchpad_space_template_id TYPE c LENGTH 35,
+
+    END OF ty_assignment,
+
+    "! <p class="shorttext">Business Role Template Launchpad Space Template Assignment</p>
+    "! Business role template launchpad space template assignment
+    BEGIN OF ty_main,
+
+      "! <p class="shorttext">Format Version</p>
+      "! Format version
+      "! $required
+      format_version TYPE zif_aff_types_v1=>ty_format_version,
+
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header         TYPE zif_aff_types_v1=>ty_header_60_cloud,
+
+      "! <p class="shorttext">General</p>
+      "! General
+      "! $required
+      assignment     TYPE ty_assignment,
```

Comment:
What do you want to emphasise. Do developers specify "general information" or the "assignments" of the object?

Reply:
We compared our editor with other editors of us and we always have this "General" at top in other editors.
But you are right developers specify more the "assignments" of the object, so "Assignment" would be better?

Reply:
Yes, "General Information" is used, if we are not able to find a better semantic name. If there is a more specific name, we suggest to use this one. See second-last paragraph of https://github.com/SAP/abap-file-formats/blob/main/docs/json.md#writing-json-schema-with-abap-types

I am not sure, maybe, it's even "Assignements" because there are multiple assignments (to the business role template and to the space template)? It's not an assignment between business role template and space template, is it? What do you think?

Reply:
I think general information would be the best choice, the whole object is an assignment of a business role template and a space template, so if it would be "Assignment(s)" this would be a duplicate and makes not that much sense in this position.